tests/custom/src/TaskRunner/Commands/TestCommands.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace My\Custom\TaskRunner\Commands;
+
+use OpenEuropa\TaskRunner\Commands\AbstractCommands;
+use Symfony\Component\Console\Input\InputOption;
+
+/**
+ * Class TestCommands.
+ */
```

Comment:
Maybe  `Commands for testing TaskRunner.` ?

Reply:
Fixed.